cypress/e2e/API/api_EngageSphere.cy.js

```diff
@@ -0,0 +1,160 @@
+/// <reference types="cypress" />
+
+//DÚVIDAS!!!
+//to.be.at.greaterThan(9999).and.to.be.lessThan(50000)
+//Retorna que o customer Littel  tem 10.000 funcionários e também que ele possuí menos de 50.000
+
+describe('ENGAGE SPHERE API TESTING', () => {
+
+    const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}`
```

Comment:
Aqui devia ser assim:

```suggestion
    const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}/customers`
```

Do jeito que está é só a URL da API, não do _endpoint_ `/customers`.

Fazendo da forma que estou sugerindo, você não precisa repetir a _string_ `/customers` em vários testes.

O que acha?

Reply:
Ao fizer isso, lembre-se de atualizar os testes que usam tal variável.